saleor/order/utils.py

```diff
@@ -363,6 +343,7 @@ def add_variant_to_order(
     channel = order.channel
 
     if line_data.line_id:
+        # It means there is an update of the order line with new quantity
```

Comment:
It is good to have additional context, but that better would be to have self-explaining code. So instead of an additional comment, we could have:
```python
is_new_line = not line_data.line_id
if not is_new_line:
    ...
```

Reply:
Make sense, thanks